scripts/Packages/Package_DFN_QFN/size_definitions/dfn.yaml

```diff
@@ -76,6 +76,33 @@ DFN-6-1EP_2x1.8mm_P0.5mm_EP1.2x1.6mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-6-1EP_3x2mm_P0.5mm_EP1.65x1.35mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/ltc-legacy-dfn/(DCB6)%20DFN%2005-08-1715%20Rev%20A.pdf'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x: 3
+  body_size_y: 2
+  lead_width: 0.25
+  lead_len: 0.2
+  EP_size_x: 1.65
+  EP_size_y: 1.35
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+  heel_reduction: 0.1 #for relatively large EP pads (increase clearance)
```

Comment:
This might not be required. set it to 0 an measure the clearance between the exposed pad (the large one in the middle) and the normal pads. If the clearance is smaller than 0.2mm then the heel reduction is needed.

Reply:
Changed, thks.